doc/api/esm.md

```diff
@@ -912,16 +912,13 @@ import packageMain from 'commonjs-package'; // Works
 import { method } from 'commonjs-package'; // Errors
 ```
 
-### `import()` expressions
+It is also possible to
+[import an ES or CommonJS module for its side effects only][].
 
-Dynamic `import()` is supported in both CommonJS and ES modules. It can be used
-to include ES module files from CommonJS code.
+### `import()` expressions
 
-```js
-(async () => {
-  await import('./my-app.mjs');
-})();
-```
+[Dynamic import][] is supported in both CommonJS and ES modules. It is the
+_only_  way to import from an ES module in CommonJS code.
```

Comment:
```suggestion
[Dynamic `import()`][] is supported in both CommonJS and ES modules. It can be used
to include ES module files from CommonJS code.
```

Technically it’s not the only way; there are `data:` URIs, `Worker`s (I think) and possibly other methods.